app/en/home/glossary/page.mdx

```diff
@@ -199,9 +200,7 @@ MCP gateways are a feature of the Arcade Engine that allows you to add and combi
 
 ### Arcade MCP (Server Development Kit)
 
-Arcade MCP is a framework for building secure MCP servers. It is responsible for the lifecycle of tool executions, and for enforcing security and authorization decisions within the boundary of the server.
-
-_Learn more about the Arcade MCP framework [here](/references)._
+Arcade MCP is the secure framework for building MCP servers. It is responsible for the lifecycle of tool executions, and for enforcing security and authorization decisions within the boundary of the server.
```

Comment:
```suggestion
`arcade-mcp` is the secure framework for building MCP servers. It is responsible for the lifecycle of tool executions, and for enforcing security and authorization decisions within the boundary of the server.
```

I think we [decided](https://arcade-ai.slack.com/archives/C06PXUCCTEX/p1762452857785579?thread_ts=1762391211.338749&cid=C06PXUCCTEX) to write the name of the package like `arcade-mcp`